ui/app.js

```diff
@@ -34,6 +34,8 @@ const dev = process.env.NODE_ENV !== 'production';
 const nextApp = next({ dev });
 const debug = require('debug')('AthenzUI:server:app');
 
+process.env.NEXT_PUBLIC_ONCALL_URL = appConfig.onCallUrl || appConfig.serverURL;
```

Comment:
apologies for the broken review, didn't notice it before
we do not have onCallUrl or serverURL in default-config.js.